configure.ac

```diff
@@ -47,23 +47,31 @@ AC_SUBST([LIBEXECDIR])
 
 AC_CONFIG_FILES([Makefile dist/tpm2-totp.pc dist/dracut/module-setup.sh dist/initcpio/install/plymouth-tpm2-totp])
 
-AX_ADD_COMPILER_FLAG([-std=c99])
-AX_ADD_COMPILER_FLAG([-Wall])
-AX_ADD_COMPILER_FLAG([-Wextra])
-AX_ADD_COMPILER_FLAG([-Wformat-security])
-AX_ADD_COMPILER_FLAG([-Werror])
-AX_ADD_COMPILER_FLAG([-fstack-protector-all])
-AX_ADD_COMPILER_FLAG([-fpic])
-AX_ADD_COMPILER_FLAG([-fPIC])
-
-# work around GCC bug #53119
-#   https://gcc.gnu.org/bugzilla/show_bug.cgi?id=53119
-AX_ADD_COMPILER_FLAG([-Wno-missing-braces])
-
-AX_ADD_LINK_FLAG([-Wl,--no-undefined])
-AX_ADD_LINK_FLAG([-Wl,-z,noexecstack])
-AX_ADD_LINK_FLAG([-Wl,-z,now])
-AX_ADD_LINK_FLAG([-Wl,-z,relro])
+AC_ARG_ENABLE([defaultflags],
+              [AS_HELP_STRING([--disable-defaultflags],
+                              [Disable default preprocessor, compiler, and linker flags.])],
+              [enable_defaultflags=$enableval],
```

Comment:
`[enable_defaultflags=$enableval]` happens by default anyway, so you could leave the third argument empty instead, see https://github.com/tpm2-software/tpm2-tss/commit/1127302cc18f1ff6bbd3f80284ceb26f13f80223#diff-67e997bcfdac55191033d57a16d1408aL254.

Reply:
Correct. I copied it as-is as I did not want to make more noise... many fixups can be done for the autoconf usage :)

I will submit a fix, thanks!